bash_scripting_101/bash_scripting_101.md

```diff
@@ -37,241 +39,443 @@ script: https://kit.fontawesome.com/83b2343bd4.js
+
+- The fixed path to the `data` directory is `~/project/data`, so `cd ~/project/data` will get you there.
 
-The `sort` command is used to categorize files in bash.
+- Relative to your current location, you can go up one level with `..` to the `notes` directory, then up another level with `..` to `project`. Once in the `project` directory you can move down a level into `data`.
 
+    * You can do this in three steps: `cd ..` followed by `cd ..` again and lastly `cd data`
+
+  * You can link the three steps together with `cd ../../data`
 </div>
-***********************************************************************
+***
 
-### Syntax
+## Editing Files
 
-- `>` takes the output of the command you executed in the terminal and places it in a new file
-- `|` takes the output of one command and passes it to the next command in the sequence. Allows for integrating of commands
-- `~` shortcut for your home directory
-- `.` shortcut for your current directory
-- `..` shortcut for your previous directory
-  The above three shortcuts are highly useful for executing scripts in the proper location once you are collaborating in an active project with multiple files and folders.
-- `*` character sequence wildcard. Very useful in running a command on all files with a given file extension as demonstrated in the mv example above.
-- `$` used to define a variable expression as used in the echo example above.
+The bash shell can see and move files of all types, but it is most useful for interacting directly with plain text files.
 
-### Knowledge Check 3
+A [**plain text**](https://en.wikipedia.org/wiki/Plain_text) file is a file that contains only text characters like the ones you can type directly using your keyboard. For example the words "plain text" at the beginning of this paragraph, which are both bolded and hyperlinked, are not being presented to you as plain text. You might have run into problems with files that aren't in plain text if, for example, you ever opened a file in a new program and lost some of the formatting like particular fonts in Word or conditional highlighting in Excel.
 
-What character passes the output of one command to the next command in a sequence?
+<div class = "learnmore">
+Even though the words "plain text" in the previous paragraph have some non-text attributes, the are all recorded as plain text in the Markdown (`.md`) file of this lesson. If you open the [Markdown file](link/to/git/of/this/file), you will see only characters that can be typed by a keyboard:
+
+```
+[**plain text**](https://en.wikipedia.org/wiki/Plain_text)
+```
+</div>
+
+If you have been using a particular program to write and edit your files like Word, RStudio, or Jupyter notebook, you will need to continue to use that program to edit those files. If you expect your project to grow substantially in size or number of files, having as much of your work as possible in plain text files may make scaling up easier.
```

Comment:
"If you have been using a particular program to write and edit your files like Word, RStudio, or Jupyter notebook, you will need to continue to use that program to edit those files."

Why?

Reply:
I guess what I'm getting at is you can always change your tooling -- if I was using RStudio I can actually switch over and write in Atom, etc., but I think I get your point.  Special file types tend to lock you in...